packages/frontend/scss/chat/_chat-list-item.scss

```diff
@@ -360,4 +361,9 @@
 #new-chat-button {
   right: 15px;
   position: absolute;
+  outline-offset: 2px;
+  &:focus {
+    height: 52px;
+    width: 52px;
```

Comment:
Ummm IDK about this. This shifts the button to the left a bit.

Is this supposed to be about visibility or just a fun little thing?

Reply:
For visibility. The small move should make it more aware...

Reply:
Well, outline should be enough I think.

Reply:
Alright then. Also consider `scale`.